ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -192,3 +192,33 @@ def test_numpy_rfftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.rfft2",
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_num_dims=st.integers(min_value=0),
+        max_num_dims=2,
+        force_tuple_axis=True,
+    ),
+    norm=st.sampled_from(["backward", "ortho", "forward"]),
+    s=st.one_of(st.lists(st.integers()), st.none()),
+    axes=st.one_of(st.none(), st.lists(st.integers())),
```

Comment:
This `axes` would not be needed since you are generating `axis` from `dtype_input_axis` above